core/models.py

```diff
@@ -97,3 +97,18 @@ class Card(models.Model):
 
     def __str__(self):
         return "%s - %s (%s)" % (self.user.username, self.card_number, self.comment)
+
+
+class NfcCard(models.Model):
```

Comment:
Så lenge man uansett ikke kan mappe kort nr og uid mot hverandre(man må
isåfall gjøre det manuelt) så mener jeg at det enda er ryddigere med 2
modeller. Å knekke krypteringen på uio kortene var KM sterk imot.
On Jul 4, 2016 10:08 PM, "nicolaei" <notifications@github.com> wrote:

> In core/models.py
> <https://github.com/cybrairai/internsystem/pull/35#discussion_r69488943>:
>
> > @@ -97,3 +97,18 @@ class Card(models.Model):
> >
> >      def __str__(self):
> >          return "%s - %s (%s)" % (self.user.username, self.card_number, self.comment)
> > +
> > +
> > +class NfcCard(models.Model):
>
> Problemet er at man ikke kan få kortnummeret ved å scanne kortet, som er
> grunnen til at jeg endte opp med å bruke uid-en. I tillegg har man da også
> mulighet til å ha egne kaffekort.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/cybrairai/internsystem/pull/35/files/1c1a03711976c58d4b4c696be7351ff5ccc904da#r69488943>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAm3G58pSg-MkUtlLtPKFclyPQ8vLz9gks5qSWhTgaJpZM4JEhQt>
> .
>